be/src/olap/comparison_predicate.h

```diff
@@ -199,6 +201,101 @@ class ComparisonPredicateBase : public ColumnPredicate {
         _evaluate_bit<true>(column, sel, size, flags);
     }
 
+    bool evaluate_and(const std::pair<WrapperField*, WrapperField*>& statistic) const override {
+        if (statistic.first == nullptr || statistic.second == nullptr) {
```

Comment:
Does we can move those check out of this function? looks like a lot of duplicate code